hamocc/mo_param_bgc.F90

```diff
@@ -128,6 +129,14 @@ module mo_param_bgc
           & mufn2o_sed,POM_remin_q10_sed, POM_remin_Tref_sed,bkox_drempoc_sed,   &
           & max_limiter
 
+  ! Time variables
+  public :: sec_per_year,sec_per_day,days_per_year
+  !********************************************************************
+  ! Time parameters
+  !********************************************************************
+  real, parameter :: sec_per_day = 86400. ! seconds per day
+  real, protected :: sec_per_year         ! seconds per year
+  real, protected :: days_per_year        ! simulated days per year
```

Comment:
Can we set default values also for `sec_per_year` and `days_per_year`, such that something is used even if `ini_bgctimes` is not called?

Reply:
Propose some - based on a 365 year?

Reply:
Thus far, we're rather free to start off with some idea, how to best handle it in the future. In any case, `ini_bgctimes` is called early in in the initialization by purpose...

Reply:
I.e. I only see very minor issues in the parameter definition - where a few `365` are hard-coded (while I am not seeing a need to change this).

Reply:
> Propose some - based on a 365 year?

Yes, that would make sense.